apps/wallet/src/shared/experimentation/features.ts

```diff
@@ -1,11 +1,21 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { GrowthBook } from '@growthbook/growthbook';
 import Browser from 'webextension-polyfill';
 
 import { API_ENV } from '_src/shared/api-env';
 
-import type { GrowthBook } from '@growthbook/growthbook';
+const GROWTHBOOK_API_KEY =
+    process.env.NODE_ENV === 'production'
+        ? 'sdk-lJ5zaQ6WI9uPth6C'
+        : 'sdk-iUMYcob41m3pnAK';
+
+export const growthbook = new GrowthBook({
+    apiHost: 'https://cdn.growthbook.io',
+    clientKey: GROWTHBOOK_API_KEY,
+    enableDevMode: true,
```

Comment:
was this intentional?

Reply:
It was, it makes it compatible with dev tools and doesn't add any overhead, so I figure it's useful to keep on.